### PR TITLE
Decouple METS Adapter deployment from SSM

### PR DESCRIPTION
**Pull request by kenoir**
Move the METS adapter to the new deployment schedule.

### CODE REVIEW

File: infrastructure/modules/worker/main.tf [hunk 1]
```diff
@@ -1,5 +1,9 @@
+locals {
+  deployment_service_name = var.deployment_service_name == "" ? var.name : var.deployment_service_name
```
Comment (jamesgorrie) [inline infrastructure/modules/worker/main.tf:2]: Is there a reason to not just make this compulsory? I sometimes get lost where things get magically created, and have to go digging them out.

File: infrastructure/modules/worker/main.tf [hunk 2]
```diff
@@ -10,10 +14,17 @@ module "service" {
   desired_task_count             = var.desired_task_count
   security_group_ids             = var.security_group_ids
   use_fargate_spot               = var.use_fargate_spot
+
+  propagate_tags = "SERVICE"
+
+  tags = {
+    "deployment:service" : local.deployment_service_name
+    "deployment:env" : var.deployment_service_env
```
Comment (alexwlchan) [inline infrastructure/modules/worker/main.tf:22]:
```suggestion
    "deployment:service" = local.deployment_service_name
    "deployment:env"       = var.deployment_service_env
```

File: makefiles/functions.Makefile [hunk 3]
```diff
@@ -82,9 +82,9 @@ endef
 define publish_service_ssm
 	$(ROOT)/docker_run.py \
     	    --aws --dind -- \
-                wellcome/weco-deploy:2.0.0  \
+                wellcome/weco-deploy:3.1.0  \
                 --project-id="$(2)" \
-                --role-arn="$(DEV_ROLE_ARN)" \
+                -v \
```
Comment (alexwlchan) [inline makefiles/functions.Makefile:87]: What does this `-v` flag do?

Comment (jamesgorrie) [reply makefiles/functions.Makefile:87]: [This](https://github.com/wellcomecollection/weco-deploy/blob/master/src/deploy/deploy.py#L66)?

File: infrastructure/modules/worker/main.tf (hunk 1 above)
Comment (kenoir) [reply infrastructure/modules/worker/main.tf:2]: This lets you override the default service name - which is necessary where the service name will not match an expected static pattern (catalogue-DATE services are an example of this).

File: makefiles/functions.Makefile (hunk 3 above)
Comment (kenoir) [reply makefiles/functions.Makefile:87]: Verbosity! I think in CI this process should be verbose for easier debugging, at least right now.

File: infrastructure/modules/worker/main.tf (hunk 1 above)
Comment (jamesgorrie) [reply infrastructure/modules/worker/main.tf:2]: Ah ok, so it's expected the name is what we want, and this is to override.

`deployment_service_name_override` or a comment?  I would feel like I should give this if coming in fresh

**Pull request merged**